.github/workflows/main_workflow.yml

```diff
@@ -183,7 +179,6 @@ jobs:
           tar -xf tendermint.tar.gz
           sudo mv tendermint /usr/local/bin/tendermint
           # pull node image needed for the integration tests
-          docker pull node:16.7.0
           docker pull trufflesuite/ganache:beta
```

Comment:
you need to pull the two other images (same mentioned on readme)

Reply:
Oh, I didnt include them as I thought for a template we wouldn't be including that. Fixing now.